src/index.html

```diff
@@ -9,6 +9,121 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
-</body>
-</html>
+  <h1>Universal Declaration of Human Rights</h1>
+    <h2>Preamble</h2>
+    <p>Whereas recognition of the inherent dignity and of the equal and inalienable
```

Comment:
1. fix indentations
2. when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear